my-terraform-project/main.tf

```diff
@@ -18,8 +18,8 @@ resource "azurerm_resource_group" "example" {
 
 resource "azurerm_storage_account" "example" {
   name                     = "examplestorageacc"
-  resource_group_name      = azurerm_resource_group.example.name
+  resource_group_name      = some_resource_group_name.example.name
```

Comment:
It seems like `some_resource_group_name.example.name` is used here, but it should likely be `azurerm_resource_group.example.name` to correctly reference the resource group defined earlier in the file.